### DISCUSSION TITLE
Flywheel units?

### DISCUSSION THREAD

**Discussion by PChild** [team401/2024-Robot-Code#105] [General]
https://github.com/team401/2024-Robot-Code/blob/f9cb6fc6b07703c405f4ac4dd2c5a3e3d297966d/src/main/java/frc/robot/subsystems/scoring/ShooterIOTalon.java#L67

Should this be in rotations per second instead of RPM?

[CTRE docs](https://v6.docs.ctr-electronics.com/en/latest/docs/migration/migration-guide/closed-loop-guide.html#closed-loop-setpoints) list rot/sec as the velocity setpoint unit for Phoenix 6.

**Comment by honzikschenk** [comment]
Good catch, just added this fix to #104
